### PR TITLE
hero test implementation details

### PR DESCRIPTION
**Pull request by mgheorghe**
new document that explains how the Hero test is implemented.
moved hero test out of draft
explains:
- testbed
- configuration
- results

### CODE REVIEW

File: documentation/general/program-scale-testing-requirements/hero-implementation-details.md [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+
+### Putting everything together
+
+#### Cables
+
+The most economical option is to use DAC cables, although optical fibers will be needed if the equipment is in separate racks. Breakout cables 1x400G to 4x100G are also used to connect the UHD400C to Novus and CloudStorm.
+
+#### Layer 1 considerations
+
+- Device port speeds for DASH are 100G or 200G or 400G, PAM4 or NRZ are UHD400C device port speeds are 100G or 200G or 400G, PAM4 or NRZ so far the 2 should interface with no issues.
```
Comment (budgrise) [inline documentation/general/program-scale-testing-requirements/hero-implementation-details.md:52]: maybe "are UHD400C" should be "and UHD400C" ?

File: documentation/general/program-scale-testing-requirements/hero-implementation-details.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+for UDP we use random source/destination ports and we set rate to 100K PPS and for 32M flows it should work fine for 320 seconds. (32M flows / 100K PPS = it will take 320 sec for all flows to receive 1 packet and be inserted into the flow table)
+
+we see here if the flow table can be filed to desired level.
+
+one item to note here is to characterize what happens when the flow table is full. will it crash? will it drop anything after? will all the extra packets be processed as slow path?
+
+### Background UDP flows
+
+
+### Background UDP flows
```
Comment (budgrise) [inline documentation/general/program-scale-testing-requirements/hero-implementation-details.md:226]: Duplicate heading

**Comment by KrisNey-MSFT** [comment]
Let's talk about this one in the call tomorrow @mgheorghe ?

**Comment by KrisNey-MSFT** [comment]
All agreed in DASH Community meeting to merge this week, no add'l comments

**Pull request merged**